### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by Nadine2016**
# Description

## Docs changes

# Context

## Notes for reviewers

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: projects/gateway/api/v1/route_table.proto [hunk 1]
```diff
@@ -172,10 +172,10 @@ message RouteTable {
     // The list of routes for the route table
     repeated Route routes = 1;
 
-    // When a delegated route defines a `RouteTableSelector` that matches multiple route tables, Gloo will inspect this
+    // When a delegated route in a VirtualService or RouteTable defines a `RouteTableSelector` that matches multiple route tables, Gloo will inspect this
     // field to determine the order in which the route tables are to be evaluated. This determines the order in which
     // the routes will appear on the final `Proxy` resource. The field is optional; if no value is specified, the weight
-    // defaults to 0 (zero).
+    // defaults to 0 (zero). Note that this field is irgnored if it is not used in a route delegation scenario.
```
Comment (artberger) [inline projects/gateway/api/v1/route_table.proto:178]: typo `irgnored` --> `ignored`

**Review by artberger — APPROVED** [review]

**Review by sam-heilbron — APPROVED** [review]

**Pull request merged**